sql/accounts_banned.sql

```diff
@@ -3,7 +3,7 @@
 -- http://www.phpmyadmin.net
 --
 -- Serveur: localhost
--- G�n�r� le : Ven 24 Juin 2011 � 07:58
+-- G�n�r� le : Ven 24 Juin 2011 � 07:58
```

Comment:
small encoding problem I think ^^

Reply:
fronch unaycode err-or say hi

Reply:
Do we even need these comments? They're literally 10 years old and serve no purpose, not even crediting the dumper.

As a side note, will the way I've exported actually affect anything, apart from these unicode comments?